technologies/javascript.md

```diff
@@ -71,10 +71,7 @@ function loadConfigFile(filePath, callback) {
 
 ### Optimise For Reading
 
-Code is written once and read many times, using abbreviations or single-character variables saves time in the short term but there's an overhead every time somebody has to read that code.
-
-Reading code is difficult enough at the best of times. Don't make it harder; it's better to have RSI in your over-worked fingers than for all your colleagues to hate you.
-
+Although normally code is written once, it's likely other colleagues maintain your contributions over time; so aim to make any code you produce self descriptive, rather than using cryptic expressions, argument or variable names.
```

Comment:
This is good, but I think we should keep the explicit example, so maybe:

... rather than using cryptic expressions, argument or variable names. For example, using abbreviations or single-character variables saves time in the short term but there's an overhead every time somebody has to read that code.